packages/transformer/src/IModelImporter.ts

```diff
@@ -597,9 +638,28 @@ export class IModelImporter implements Required<IModelImportOptions> {
       options: this.options,
       targetDbId: this.targetDb.iModelId || this.targetDb.nativeDb.getFilePath(),
       doNotUpdateElementIds: CompressedId64Set.compressSet(this.doNotUpdateElementIds),
+      duplicateCodeValueMap: this._duplicateCodeValueMap ? Object.fromEntries(this._duplicateCodeValueMap) : undefined,
       additionalState: this.getAdditionalStateJson(),
     };
   }
+
+  public resolveDuplicateCodeValues(): void {
+    if (!this.handleElementCodeDuplicates || !this._duplicateCodeValueMap)
+      return;
+
+    for (const [elementId, codeValue] of this._duplicateCodeValueMap) {
+      const element = this.targetDb.elements.getElement(elementId);
+      element.code.value = codeValue;
+      element.update();
+    }
+  }
+
+  private trackCodeValueDuplicate(targetElementId: Id64String, codeValue: string) {
+    if (!this._duplicateCodeValueMap)
+      this._duplicateCodeValueMap = new Map<Id64String, string> ();
```

Comment:
why make this possibly undefined? An empty map is cheap

Reply:
removed nullable attribute